dev_tools/check_useless_exclude_paths_hooks.py

```diff
@@ -101,10 +112,22 @@ def have_non_existent_paths_or_duplicates(hooks_list: list[Any]) -> bool:
     return bool(non_existing_paths or duplicates)
 
 
+def print_excluded_files_count(hooks_list: list[Any]) -> None:
+    total_excluded = sum(hook.count_excluded_files() for hook in hooks_list)
+    print(f"Total number of excluded files: {total_excluded}")
+
+    for hook in hooks_list:
```

Comment:
I think this is not necessarily sorted alphabetically by hook name, right? Shall we sort to get a stable output?

Reply:
The order is based on the pre-commit config. To me this sounds like a valid approach and might be helpful to find the hooks in the config.